sql/src/main/java/io/crate/protocols/postgres/types/PGTypes.java

```diff
@@ -75,9 +75,11 @@
         .put(new SetType(DataTypes.BOOLEAN), PGArray.BOOL_ARRAY)
         .put(new SetType(DataTypes.TIMESTAMP), PGArray.TIMESTAMPZ_ARRAY)
         .put(new SetType(DataTypes.STRING), PGArray.VARCHAR_ARRAY)
+        .put(new SetType(DataTypes.IP), PGArray.VARCHAR_ARRAY)
         .put(new SetType(DataTypes.GEO_POINT), PGArray.FLOAT8_ARRAY)
         .put(new SetType(DataTypes.GEO_SHAPE), PGArray.JSON_ARRAY)
         .put(new SetType(ObjectType.untyped()), JsonType.INSTANCE)
+        .put(DataTypes.GEO_POINT, PGArray.FLOAT8_ARRAY)     // register last to map PGArray.FLOAT8_ARRAY to a array(double)
```

Comment:
Maybe change the comment to something like this. I first didn't get how the ordering is relevant.

```suggestion
        .put(DataTypes.GEO_POINT, PGArray.FLOAT8_ARRAY)     // Must come after array(double) -> float8-array mapping to avoid override in the static block further below
```